src/flowMC/resource/model/nf_model/base.py

```
@@ -56,7 +56,9 @@ def __call__(
         return self.forward(x)
 
     @abstractmethod
-    def log_prob(self, x: Float[Array, " n_dim"]) -> FloatScalar:
+    def log_prob(
+        self, x: Float[Array, " n_dim"] | Float[Array, "n_sample n_dim"]
```

Ruff (0.16.3)

[error] Syntax error in forward annotation: Expected an expression

(F722)